mrr/src/main/java/gov/cdc/dex/mrr/TimerTriggerFunction.kt

```diff
@@ -77,42 +80,39 @@ import java.time.LocalDateTime
                 context.logger.info("Cache Response : " + jedis.ping())
                 val mmgaClient = MmgatClient()
                 context.logger.info("STARTING MMGATRead services")
-                var mmgaGuide = mmgaClient.getGuideAll()
+                val mmgaGuide = mmgaClient.getGuideAll()
                 // gson = new GsonBuilder().create();
                 //  mmgat = gson.toJson(sb.toString());
-                if (mmgaGuide != null) {
-                    val elem: JsonElement = parser.parse(mmgaGuide.toString())
-                    context.logger.info("Json Array size:" + elem.asJsonObject.getAsJsonArray("result").size())
-                    val mmgatJArray = elem.asJsonObject.getAsJsonArray("result")
-                    context.logger.info("Json Array size:" + mmgatJArray.size());
-                    val mmgatMap = mapOf<String, String>()
-                    val gson = GsonBuilder().create()
-
-                    for (mmgatjson in mmgatJArray) {
-                        val mj = mmgatjson.asJsonObject
-                        if (mj.get("guideStatus").getAsString().equals("UserAcceptanceTesting") || mj.get("guideStatus")
-                                .getAsString().equals("Final")
-                        ) {
-                            val id = (mj.get("id").getAsString())
-                            context.logger.info("MMGAT id:" + id)
-                            var mGuide = mmgaClient.getGuideById(id)
-                            var key = mj.get("name").getAsString()
-                            if (jedis.exists(key.toString()))
-                                jedis.del(key.toString())
-                            jedis.set(key.toString(), gson.toJson(mGuide))
-
-                        }
-                    }
+                val elem: JsonElement = parser.parse(mmgaGuide.toString())
+                context.logger.info("Json Array size:" + elem.asJsonObject.getAsJsonArray("result").size())
+                val mmgatJArray = elem.asJsonObject.getAsJsonArray("result")
+                context.logger.info("Json Array size:" + mmgatJArray.size())
+                val gson = GsonBuilder().create()
 
+                for (mmgatjson in mmgatJArray) {
+                    val mj = mmgatjson.asJsonObject
+                    if (mj.get("guideStatus").getAsString().equals("UserAcceptanceTesting") || mj.get("guideStatus")
```

Comment:
for clarity, create constants for "guideStatus" and "UserAcceptanceTesting" and "Final" and use those.
Are we sure the casing is always like that? do we need to use toUppercase() to make sure?

Reply:
done

Reply:
I'll approve this for now, but the standard way of defining constants is ALL_CAPS :)
like this (and in kotlin, it's usually inside a companion object if you want to  make those available like the old java static constants.)
 companion object {
        const val GUIDANCE_USER_ACCEPTANCE_TESTING = "UserAcceptanceTesting"
        const val GUIDANCE_FINAL = "Final"
    }